backend/src/Designer/TypedHttpClients/ImageClient/ImageClient.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Altinn.Studio.Designer.TypedHttpClients.ImageClient;
+
+public class ImageClient
+{
+    private readonly HttpClient _httpClient;
+
+    public ImageClient(HttpClient httpClient)
+    {
+        _httpClient = httpClient;
+    }
+
+    public async Task<HttpResponseMessage> ValidateUrlAsync(string url)
```

Comment:
It is better to create a dedicated result class than to return null or HttpResponseMessage.
For example `ImageValidationResult`. You can start by having only one `IsSuccess` property.